packages/ui/src/components/Badge/Badge.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+export default defineComponent({
+  name: 'SBadge',
+})
+</script>
+
+<script setup lang="ts">
+import { SSpinner } from '../Spinner'
+import { BadgeType } from './types'
+
+const props = withDefaults(
+  defineProps<{
+    type?: BadgeType
+    colorBackground?: boolean
+    withBorder?: boolean
+    onlyMarker?: boolean
+  }>(),
+  {
+    type: 'active',
+    colorBackground: false,
+    withBorder: false,
+    onlyMarker: false,
+  },
+)
+
+const showSpinner = computed(() => {
+  return props.type === 'pending'
+})
+</script>
+
+<template>
+  <div
+    tabindex="0"
+    :class="[
+      's-badge',
+      { 's-badge--border': withBorder },
+      colorBackground ? `s-badge--color-${type} text-white` : ' text-brand-pms-black',
+    ]"
+  >
+    <div class="flex items-center justify-start">
+      <div
+        v-if="!showSpinner"
+        class="marker"
+        :class="!colorBackground ? `s-badge--color-${type}` : 'bg-white'"
+      />
+      <SSpinner
+        v-else
+        class="spinner"
+      />
+      <span
+        v-if="!onlyMarker"
+        class="title"
+      ><slot /></span>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+@use '@/theme';
+.s-badge {
+  @apply cursor-default select-none block;
+  border-radius: 30px;
+  padding: 5px 10px;
+  font-size: 10px;
+  font-family: Sora;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 140%;
+  width: fit-content;
+  letter-spacing: 0.06em;
+  font-feature-settings: 'case' on;

```

Comment:
Looks like you forget to use typography token correctly. Please see README at the theme package.